docs/source/deployment/nodes_grouping.md

```diff
@@ -0,0 +1,107 @@
+# Node Grouping in Kedro: Pipelines, Tags, and Namespaces
+
+When deploying a Kedro project, grouping nodes effectively is crucial for maintainability, debugging, and execution control. This document provides an overview of three key grouping methods: Pipelines, Tags, and Namespaces, along with a list of their strengths, limitations, best use cases, and links to how to use them.
+
+# Choosing between pipelines, tags, and namespaces in Kedro
+
+## Nodes grouping using pipelines in Kedro
+
+### What works with Kedro
+- If your project contains different pipelines, you can use them as predefined node groupings for deployment.
+- Pipelines can be executed separately in the deployment environment.
+- With the visualisation in Kedro Viz, you can switch to see different pipeline in a more isolated view
+<br>
+![Switching between different pipelines in Kedro Viz](../meta/images/kedro_viz_switching_pipeline.gif)
+
+### What doesn't work with Kedro
+- If you need to create custom node groupings that are different with your existing pipelines, creating new pipelines for them may not be convenient. Instead, you can use alternative grouping methods such as tags or namespaces.
```

Comment:
I don't quite understand this sentence 🤔

Reply:
i think what I'm trying to say is "if you want to group nodes to something different than the current pipeline structure, instead of creating a new pipeline, using tags or namespace can help you with that"

Reply:
I'll make it clearer

Reply:
Okay, now I understand! However, it's not really a disadvantage right? It's just that users have an alternative in this case.